osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs

```diff
@@ -76,6 +76,15 @@ public override Color4 AccentColour
             }
         }
 
+        public void HideButApproachCircle()
```

Comment:
Maybe it's an idea to expose the parts you want to have influence on rather than having mod specific behaviour in the `DrawableHitCircle`.